docs/10_Overview_and_Usage.md

```diff
@@ -1026,6 +1027,42 @@ curl --location --request DELETE 'url.mia-platform.eu/v2/books/?category=sci-fi&
 The endpoint always responds `200 OK`, with an integer number in the body representing the count of deleted documents. 
 The response is 200 also when no documents are found, in that case the count will be 0.
 
+### Import
+
+You can import element into a collection from a file csv, json or ndjson.
+
+#### Import insert
+
+To import all the elements of a file you have to use the `POST` method. Mind that the method will fail if inside the file you have documents with an `_id` already present in your collection. To avoid this refer to the [upsert method](#import-upsert).
+
+The route to call is the following:
+
+`POST` `https://your-url/<CRUD collection endpoint>/import`
+
+Below you can see an example:
+
+```shell
+curl --data "file=@/path/to/file.json" -H 'Content-Type: multipart/form-data;boundary=XXXXX' -X POST http://url.mia-platform.eu/v2/books/import
+```
+
+When successful the endpoint responds `201 OK`, with the text "File uploaded successfully"
+
+#### Import upsert
+
+You can also import all the elements from a file using an `upsert` strategy, meaning that all the documents in the file having an already present `_id` in your collection will be updated with the value of the document in the file. Of course, if the document in the file does not have an `_id` present in the collection it will be inserted as a new document.
```

Comment:
```suggestion
You can also import all the elements from a file using an `upsert` strategy, meaning that all the documents in the file having an already present `_id` field in your collection will be updated with the value of the document in the file.  
In case a document in the file does not provide the `_id` field, then the import procedure will use the input document as filter to look for an existing document with the same data in the collection. If the document matches a collection record then, the metadata (e.g. `updatedAt` field) of existing document is updated. Otherwise a new document will be inserted in the collection```

Does this clarify the new behavior?